### DISCUSSION TITLE
why on inspect, the drawer pops up for short duration how to fix? URGENT

### DISCUSSION THREAD

**Discussion by Vaibhav227** [saadeghi/daisyui#2133] [Q&A]
why on inspect, the drawer pops up for short duration how to fix?

**Comment by saadeghi** [comment]
What do you mean on inspect?

**Reply by Vaibhav227** [reply]
developer tools/console

**Reply by Vaibhav227** [reply]
on opening inspect on webpage it happens,

**Reply by saadeghi** [reply]
Would you mind sharing more info?  
- Share a code example
- If it's a visual issue, share a video as well
- Your browser name/version
- What version of daisyUI are you using?

I don't see the drawer when I open devtools so please share more info so I can see the problem. It's hard to guess what is the issue when you don't provide any details.

**Reply by Vaibhav227** [reply]
The drawer pops up for small duration with devtools open on mutliple reloads.
I m using daisy ui version 3  and latest chrome.

**Reply by saadeghi** [reply]
❌ Share a code example
❌ If it's a visual issue, share a video as well
✅ Your browser name/version
✅ What version of daisyUI are you using?

**Reply by Vaibhav227** [reply]
https://github.com/saadeghi/daisyui/assets/51129176/945907ea-3c21-4c07-b5ed-69ff67a47e3d

**Reply by writeDavid** [reply]
Can you show an example, that isn't daisyui.com?  This could be just an implementation issue with daisyui.com and not the underlying drawer component.
